druid-shell/Cargo.toml

```diff
@@ -14,7 +14,9 @@ rustdoc-args = ["--cfg", "docsrs"]
 default-target = "x86_64-pc-windows-msvc"
 
 [features]
-x11 = ["x11rb", "nix", "cairo-sys-rs"]
+x11 = ["x11rb", "nix", "cairo-sys-rs", "cairo-rs"]
+GTK = ["cairo-rs", "gio", "gdk", "gdk-sys", "glib", "glib-sys", "gtk-sys", "gtk"]
```

Comment:
can we just call this `gtk`? It's okay if this collides with the `gtk` crate I believe.

Reply:
I also prefer just calling it `gtk`, but Cargo does not agree with us :(

(It will error about two features with the same name)

Reply:
what about `platform-gtk`?

Reply:
Okay, what I propose:

in `druid-shell/Cargo.toml`, rename the *actual* gtk crate to `gtk-rs`, and then call the *feature* `gtk`:

```toml
[features]
x11 = ["x11rb", "nix", "cairo-sys-rs", "cairo-rs"]
gtk = ["cairo-rs", "gio", "gdk", "gdk-sys", "glib", "glib-sys", "gtk-sys", "gtk-rs"]
default = ["gtk"]

[target.'cfg(target_os="linux")'.dependencies]
# other deps here
gtk-rs = { version = "0.9.2", features = ["v3_22"], package = "gtk", optional = true }
```

Reply:
I think that did the job :+1: